src/components/MyAccountUserInfo/MyAccountUserInfo.js

```diff
@@ -1,10 +1,7 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import SkeletonWrapper from 'components/SkeletonWrapper';
-
 import { useSelector } from 'react-redux';
-import { withTranslation } from 'react-i18next';
-import labeling from 'containers/labeling';
+import { useTranslation } from 'react-i18next';
+import SkeletonWrapper from 'components/SkeletonWrapper';
```

Comment:
Why this component was added? (SkeletonWrapper)

Reply:
It was there before but import order changed